drivers/platform/ftd2xx/mpsse/ftd2xx_gpio.h

```diff
@@ -38,7 +38,13 @@
 #include "no_os_gpio.h"
 #include "ftd2xx.h"
 
+#define FTD2XX_GPIO_PIN(x)		NO_OS_BIT(x)
+
 #define FTD2XX_MAX_PIN_NB		7
+#define FTD2XX_MAX_PORT_NB		4
+
+extern uint8_t ftd2xx_gpio_pins_dir[FTD2XX_MAX_PORT_NB];
+extern uint8_t ftd2xx_gpio_pins_val[FTD2XX_MAX_PORT_NB];
```

Comment:
Why do you want to make these available in the header? Looks like an implementation detail that should only be accessible to the driver. Also, you will only allow one instance of the driver by having a single set of arrays.